mkdocs/themes/mkdocs/base.html

```diff
@@ -30,7 +30,7 @@
 
       {%- block libs %}
 
-        <script src="{{ 'js/jquery-1.10.2.min.js'|url }}" defer></script>
+        <script src="{{ 'js/jquery-3.6.0.min.js'|url }}" defer></script>
```

Comment:
Have you done any testing showing that no frontend functionality is broken and that no browser support is dropped unreasonably?